spec/helpers/application_helper_spec.rb

```diff
@@ -378,4 +378,58 @@ def current_user_can_publish? = false
       expect(rows).to include(include(key: "<span>test title</span>"))
     end
   end
+
+  describe "#publish_text" do
+    it "returns correct text if the manual is published" do
+      manual = FactoryBot.build_stubbed(:manual)
+      allow(manual).to receive(:state).and_return("published")
```

Comment:
Could we override the attributes on the factory to set the state value, rather than mocking it? Or would that depend on the database?